requirements.txt

```diff
@@ -10,3 +10,4 @@ factory-boy==2.12.0
 wagtail-factories==2.0.0
 django-cors-headers==3.0.2
 wagtail-headless-preview==0.1.4
+django_extensions==2.2.9
```

Comment:
I am not convinced we need the extra dependency. Should only be a dev dependency